packages/rspack/src/compiler.ts

```diff
@@ -78,54 +79,6 @@ class HotModuleReplacementPlugin {
 	apply() {}
 }
 
-const compilationHooksMap = new WeakMap();
```

Comment:
I just merge these classes, because this class is copied from `normalModule.js`, keeping the other thing as is. Actually, this hook is not called, so I don't think it will make a difference.